modules/xfem/src/userobjects/GeometricCutUserObject.C

```diff
@@ -262,11 +274,17 @@ GeometricCutUserObject::finalize()
 
   for (const auto & it : _marked_elems_2d)
     for (const auto & gmei : it.second)
+    {
       _xfem->addGeomMarkedElem2D(it.first, gmei);
+      _xfem->addGeomMarkerIDElem(_interface_id, it.first);
```

Comment:
Couldn't we just add these two additional parameters to `addGeomMarkedElem2D()` rather than having a second call?

Reply:
Good point. I merged these two.